ballerina/api_del_doc_with_credit_block/Ballerina.toml

```diff
@@ -1,12 +1,12 @@
 [package]
 org = "ballerinax"
-name = "sap.s4hana.delivery"
+name = "sap.s4hana.api_del_doc_with_credit_block"
 version = "1.0.0"
 distribution = "2201.9.0"
 authors = ["Ballerina"]
 keywords = ["Business Management/ERP", "Cost/Paid", "Vendor/SAP", "Sales and Distribution", "SD"]
-export = ["sap.s4hana.delivery"]
-repository = "https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.delivery"
+export = ["sap.s4hana.api_del_doc_with_credit_block"]
+repository = "https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.api_del_doc_with_credit_block"
```

Comment:
Why is the repository changed?

Reply:
Change added in [Add PR review changes to gradle build](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.delivery/pull/7/commits/031c361cb9b165454c9d51d179f5c52002682d85)